test/src/e2e_vm_tests/harness.rs

```diff
@@ -147,6 +144,8 @@ pub(crate) fn test_json_abi(file_name: &str) -> Result<()> {
         fs::read_to_string(oracle_path).expect("Something went wrong reading the file.");
     let output_contents =
         fs::read_to_string(output_path).expect("Something went wrong reading the file.");
+    println!("{}", oracle_contents);
+    println!("{}", output_contents);
```

Comment:
Do we still want to print these? Or were they just for debugging parser changes?